src/stylesheets/components/_actions.scss

```diff
@@ -0,0 +1,33 @@
+.sds-button--actions {
```

Comment:
There is a class called `sds-button--action` which defines the current (wrong) look in _menu.scss. Is there a reason that class needs to be preserved? If not, that class should be updated rather than creating two, very similar classes.

Reply:
The sds-button--action class is kept for the blue action icon. I may combine and put them together in the _actions.scss file.

Reply:
Okay so it sounds like the ask is more for a variant of the existing action icon rather than a replacement. In that case, it'd probably be better to start with the existing class and then adjust whatever styles are different with this variant(bg color, border thickness, etc). The benefit of this is that the common styles will stay in sync as additional changes are made and we don't have the buttons unintentionally drifting apart by devs missing that they need to update styles in two places.

If I'm misunderstanding and the ask is for essentially two different buttons, then it makes sense to not have common styles shared between then.

Regardless, I think the class name should be changed to something that clearly differentiates between the existing and new actions button. If it isn't, there may be more hassle when the next developer who needs an action button comes along and has to determine whether they want `.sds-button--action` or `.sds-button--actions`. It'd be easier to understand `.sds-button--action` or `.sds-button--action-light` if the new button is just a variant.